centaur/src/main/resources/standardTestCases/drs_usa_jdr_preresolve.test

```diff
@@ -0,0 +1,35 @@
+name: drs_usa_jdr_preresolve
+testFormat: WorkflowSuccess
+backends: ["papi-v2-usa"]
+skipDescribeEndpointValidation: true
+
+files {
+  workflow: drs_tests/drs_usa_jdr.wdl
+  options-dir: "Error: BA-6546 The environment variable CROMWELL_BUILD_RESOURCES_DIRECTORY must be set/export pointing to a valid path such as '${YOUR_CROMWELL_DIR}/target/ci/resources'"
+  options-dir: ${?CROMWELL_BUILD_RESOURCES_DIRECTORY}
+  options: ${files.options-dir}/papi_v2_usa_preresolve.options.json
+  inputs: drs_tests/drs_usa_jdr.inputs
+}
+
+metadata {
+  workflowName: drs_usa_jdr
+  status: Succeeded
+
+  "outputs.drs_usa_jdr.path1" =
+    "/cromwell_root/jade.datarepo-dev.broadinstitute.org/v1_f90f5d7f-c507-4e56-abfc-b965a66023fb_585f3f19-985f-43b0-ab6a-79fa4c8310fc/hello_jade.json"
+  "outputs.drs_usa_jdr.path2" =
+    "/cromwell_root/broad-jade-dev-data-bucket/e1941fb9-6537-4e1a-b70d-34352a3a7817/ad783b60-aeba-4055-8f7b-194880f37259/hello_jade_2.json"
```

Comment:
This line appears to be the significant difference between this and `drs_usa_jdr.test`:
```
    "/cromwell_root/jade.datarepo-dev.broadinstitute.org/v1_001afc86-da4c-4739-85be-26ca98d2693f_ad783b60-aeba-4055-8f7b-194880f37259/hello_jade_2.json"
```
However, I don't understand the significance of the difference. The file appears to be localized in both cases, but maybe the difference is an artifact of when/how the localization happened? I'd like to understand this better.

Reply:
Happy to review in person. 

For now, look at the paths of the downloaded files in the jdr test file. Ones that have been preresolved will have download file paths based on the “gs://“ path, while the non-preresolved paths have downloaded file paths based on the “drs://“ path. 

Also look at the “cloud” paths in the jdr test file. One has skipped localization and starts with “gs://“, the other has been localized even though the WDL asked to skip downloading and starts with “/cromwell_root/“